docs/sources/mimir/configure/configuration-parameters/index.md

```diff
@@ -1962,32 +1957,23 @@ The `ruler` block configures the ruler.
 [notification_timeout: <duration> | default = 10s]
 
 alertmanager_client:
-  # (advanced) Enable TLS for gRPC client connecting to alertmanager.
-  # CLI flag: -ruler.alertmanager-client.tls-enabled
-  [tls_enabled: <boolean> | default = true]
+  # (advanced)
+  [tls_enabled: <boolean> | default = ]
```

Comment:
Since we unbound the flags for the old ruler config fields, we also lose the docstrings. It's just moved down below under limits. That seems to be desired/expected by the process though. I chose not to publicly mark these as deprecated just yet.

Reply:
Personally I do not see a reason not to mark as deprecated. It's another place where people can be warned that it will be deprecated, and if it's going to get moved I would want as much visibility as possible